### ISSUE TITLE
This project is no longer maintained

### ISSUE THREAD

**Issue by maxep** [maxep/MXParallaxHeader#124]
Hey,

Unfortunately, I have decided to stop maintaining this project. This code is now an outdated implementation in Objective-C and I'm not giving it the attention it deserves. 

Please let me know if you want to become a contributor or feel free to share any alternative or forks in the comments of this issue.

Thank you for your understanding!

**Comment by Kedar-27** [comment]
Yes, I do have alternative solution, Please check my repo [here](https://github.com/Kedar-27/KSParallaxScrolling) Thank you.

**Comment by ngochiencse** [comment]
Hi I have fork your repo into swift version: HPParallaxHeader. Hope this could help:
https://github.com/ngochiencse/HPParallaxHeader

**Comment by ngochiencse** [comment]
> Yes, I do have alternative solution, Please check my repo [here](https://github.com/Kedar-27/KSParallaxScrolling) Thank you.

Hi I have checked, but apparently the solution is not stable enough. It still have bugs when I tried to use. You can try my solution instead:  https://github.com/ngochiencse/HPParallaxHeader

**Comment by sugitatestblue** [comment]
**Migration.**

```Podfile
-  pod 'MXParallaxHeader', '0.6.1'
+  pod 'HPParallaxHeader', '1.1.7'
```

```
-import MXParallaxHeader
+import HPParallaxHeader
```
```
-final class HogeViewController: MXScrollViewController {
+final class HogeViewController: HPScrollViewController {
```

In the `xxxx.storyboard`, 

`MXParallaxHeaderSegue` -> `HPParallaxHeaderSegue`
